Android/TursuApp/app/src/main/java/com/example/tursuapp/authentication/homepage/ui/payment/PaymentFragment.kt

```diff
@@ -89,7 +122,7 @@ class PaymentFragment : Fragment() {
 
             if (view.findViewById<TextInputEditText>(R.id.nameinputtext).text.toString() == "") {
                 Toast.makeText(context, "Lütfen geçerli bir isim giriniz.", Toast.LENGTH_SHORT).show()
-            } else if (view.findViewById<TextInputEditText>(R.id.cardnumberinputtext).text.toString().length != 16|| !isAllNumber((view.findViewById<TextInputEditText>(R.id.cardnumberinputtext).text.toString()))) {
+            } else if (view.findViewById<TextInputEditText>(R.id.cardnumberinputtext).text.toString().length != 19|| !isAllNumber((view.findViewById<TextInputEditText>(R.id.cardnumberinputtext).text.toString()))) {
                 Toast.makeText(context, "Lütfen geçerli bir kart numarası giriniz.", Toast.LENGTH_SHORT).show()
```

Comment:
You should translate Turkish sentences into English.

Reply:
Will be fixed on another branch